content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -241,21 +241,72 @@ jobs BEFORE you reference them from another component (via the image-references
+3. Open and `/hold` a pull request, PR2, against the repo of the operator which references the old operand component name in its `image-references` file. PR2 should update `image-references` to use the new component name.
+4. Open and `/hold` a pull request, PR3, against github.com/openshift/ocp-build-data in the openshift-4.x branch of the targeted release(s). Note that ART may have already cut a new release branch, meaning you need to open a PR for openshift-4.x+1 as well. The PR should change the `name` field in the component's metadata. Note that in ART metadata, the desired name should be prefixed with `ose-`. For example, the `etcd` operand component is name definition [can be seen here, for openshift-4.11](https://github.com/openshift/ocp-build-data/blob/129ef9fb8335991d88ce9f41a61057c6ab2bb08e/images/cluster-etcd-operator.yml#L23).  
+5. Submit a copy of [this ART team template](https://issues.redhat.com/browse/ART-1443) to communicate to ART that a component name change is desired. Include the PRs in the Jira ticket.
+6. PR1 and PR3 should be passing tests and ready to merge (PR2 will be failing at this point). No PRs other than PR1 and PR2 should be merged in their respective github.com repositories during the following process. In a synchronous chat with an ART team member or release manager (`@release-artists` in `#aos-art` on Slack) the following should be performed:
+   1. On the central app.ci CI cluster, a release-artist should check the current registry.ci.openshift.org image associated with the component in the `-n ocp is/4.{minor}` release image stream. For example, `oc -n ocp get istag 4.11:{old-component-name} -o=json | jq .tag.from.name` should output a pullspec like `registry.ci.openshift.org/ocp/4.11@sha256:...` for 4.11.
+   2. The release-artist should run `oc -n ocp tag {existing-registry.ci-openshift-org@sha26..} 4.{minor}:{new-component-name}` to establish a tag with the new operand component name.
+   3. Run `/retest` on PR2. Tests should now pass.
+   4. Unhold and merge PR1.
+   5. Unhold and merge PR2.
+   6. Unhold and have the release-artist merge PR3.
+   7. It is not time sensitive, but before the ART Jira ticket is closed, the release-artist must remove the old component name from CI: `oc -n ocp tag 4.{minor}:{old-component-name} -d`.
+7. Monitor the subsequent [CI nightlies on amd64 OpenShift release controller](https://amd64.ocp.releases.ci.openshift.org/#4.11.0-0.ci). Continue to do so until you see a CI nightly produced which reports the new component name in `oc adm release info <ci-nightly-release-payload-pullspec>`. If a CI nightly fails to assemble, immediately report the issue to `@team-technical-release` and `@release-artists` so that the incident can be recovered as quickly as possible.
```

Comment:
What is a `CI nightly`?  I normally think of us as having "ci payloads" and "nightly payloads".  the link goes to what i consider CI payloads.  Which do you mean here?

I see below you refer to ART nightlies, so i guess you mean what i have always called "ci payloads".  I think calling them "ci nightlies" is going to lead to confusion, especially since they are not built nightly.  People are likely to see "nightly" and think it's referring to the ART/OSBS built payloads. (i know, ART nightlies aren't nightly either, but that's the terminology we seem to have settled into from what i've seen).

Reply:
> If a CI nightly fails to assemble

how can you tell if a CI payload fails to assemble?

Reply:
Changed to `CI payload` and included information about the error to look for in the next push.